templates/data/_form-data-kafka.html

```diff
@@ -0,0 +1,113 @@
+                      <span class="p-checkbox__label" id="canonicalUpdatesOptIn">I agree to receive information about Canonical's products and services.</span>
+                    </label>
+                  </li>
+                  <li class="p-list__item">In submitting this form, I confirm that I have read and agree to Canonical's <a href="/legal/terms-and-policies/privacy-policy">Privacy Policy</a>.</li>
+                  {# These are honey pot fields to catch bots #}
+                  <li class="u-off-screen">
+                    <label class="website" for="website">Website:</label>
+                    <input name="website" type="text" class="website" autocomplete="off" value="" id="website" tabindex="-1" />
+                  </li>
+                  <li class="u-off-screen">
+                    <label class="name" for="name">Name:</label>
+                    <input name="name" type="text" class="name" autocomplete="off" value="" id="name" tabindex="-1" />
+                  </li>
+                  {# End of honey pots #}
+                </ul>
+
+                <div class="u-hide">
+                  <input type="hidden" aria-hidden="true" aria-label="hidden field" name="returnURL" value="{{ returnURL }}" />
+                  <input type="hidden" aria-hidden="true" aria-label="hidden field" name="formid" value="5505" />
+                  <input type="hidden" aria-hidden="true" aria-label="hidden field" name="Consent_to_Processing__c" value="yes" />
+                  <input type="hidden" aria-hidden="true" aria-label="hidden field" name="acquisition_url" id="acquisition_url" value="{{ request.url }}" />
+                  <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_campaign" id="utm_campaign" value="" />
+                  <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_medium" id="utm_medium" value="" />
+                  <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_source" id="utm_source" value="" />
+                  <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_content" id="utm_content" value="" />
+                  <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_term" id="utm_term" value="" />
+                  <input type="hidden" aria-hidden="true" aria-label="hidden field" id="preferredLanguage" name="preferredLanguage" maxlength="255" value="" />
+                  <input type="hidden" aria-hidden="true" aria-label="hidden field" name="productContext" id="product-context" value="{{ product }}" />
+                </div>
+
+                <div class="pagination">
+                  <button type="submit" class="pagination__link--next p-button--positive" aria-label="Submit">Let's discuss</button>
+                </div>
+              </div>
+            </div>
+          </form>
+        </div>
+
+        <div class="js-pagination js-pagination--2 u-hide">
```

Comment:
I think we may not need this section if we are using the `#success` hash, which returns to the page with a JS message. We tend to use a separate thank you modal page when there is a series of steps, like a survey with multiple steps.